explorer/client/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { DefaultRpcClient as rpc } from '../../utils/internetapi/SuiRpcClient';
+import { navigateWithUnknown } from '../../utils/searchUtil';
+import { findDataFromID } from '../../utils/static/searchUtil';
+import { trimStdLibPrefix, processDisplayValue } from '../../utils/stringUtils';
+import DisplayBox from '../displaybox/DisplayBox';
+
+import styles from './OwnedObjects.module.css';
+
+type resultType = {
+    id: string;
+    Type: string;
+    Version: string;
+    display?: string;
+}[];
+
+const DATATYPE_DEFAULT: resultType = [
+    {
+        id: '',
+        Type: '',
+        Version: '',
+        display: '',
+    },
+];
+
+function OwnedObjectStatic({ objects }: { objects: string[] }) {
+    const results = objects.map((objectId) => {
+        const entry = findDataFromID(objectId, undefined);
+        return {
+            id: entry?.id,
+            Type: entry?.objType,
+            Version: entry?.version,
+            display: entry?.data?.contents?.display,
+        };
+    });
+
+    return <OwnedObjectView results={results} />;
+}
+
+function OwnecObjectInternetAPI({ objects }: { objects: string[] }) {
```

Comment:
nit: should be `Owned` instead of `Ownec`, and use a shorter name than the "InternetAPI" suffix

Reply:
Done. I've also broadened the net and changed references to InternetAPI / internetapi to API/api.